ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -13,6 +13,9 @@ class_relationships:
 # Containing Relations
 - Endpoint(components) 1:MC OpenstackComponent
 - Instance 1:MC Vnic
+# Cinder
+- Volume 1:MC Snapshot
```

Comment:
I am not aware of moving snapshots between volumes use case, at least within openstack. In openstack, a snapshot has a volume attribute, pointing back to its original volume.